src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/CapabilitiesTest.java

```diff
@@ -261,6 +264,41 @@ public void testAttribution() throws Exception {
         assertXpathEvaluatesTo("1", "count(//Attribution/Title)", doc);
         assertXpathEvaluatesTo("1", "count(//Attribution/LogoURL)", doc);
     }
+    
+
+    @Test
+    public void testLayerGroup() throws Exception {
+        LayerInfo points = getCatalog().getLayerByName(MockData.POINTS.getLocalPart());
+        CatalogBuilder builder = new CatalogBuilder(getCatalog());        
+        
+        //create layergr
+        LayerGroupInfo lg = getCatalog().getFactory().createLayerGroup();
+        //attribution
+        lg.setName("MyLayerGroup");
+        lg.getLayers().add(points);
+        builder.calculateLayerGroupBounds(lg, CRS.decode("EPSG:4326"));
+        lg.setAttribution(getCatalog().getFactory().createAttribution());
+        lg.getAttribution().setTitle("My Attribution");
+        MetadataLinkInfo info = getCatalog().getFactory().createMetadataLink();
+        info.setType("text/html");
+        info.setMetadataType("FGDC");
+        info.setContent("http://my/metadata/link");
+        lg.getMetadataLinks().add(info);
+        getCatalog().add(lg);
+        
+        try {
+            System.out.println(getAsString("wms?service=WMS&request=getCapabilities&version=1.1.1"));
+            Document doc = getAsDOM("wms?service=WMS&request=getCapabilities&version=1.1.1", true);
+            print(doc);
```

Comment:
Handy for debugging, but nasty to have at runtime, please comment out this "print"